### PR TITLE
Control module and counting implementations

### PR DESCRIPTION
**Pull request by salvalatorre**

### CODE REVIEW

File: applications/DemStructuresCouplingApplication/python_scripts/dem_structures_coupling_gid_output.py [hunk 1]
```diff
@@ -168,6 +176,10 @@ def write_dem_fem_results(self, label):
             kratos_variable = globals()[var]
             self._write_nodal_results(label, self.rigid_faces_model_part, kratos_variable)
 
+        for var in self.contact_gauss_points_results:
+            kratos_variable = globals()[var]
```
Comment (philbucher) [inline applications/DemStructuresCouplingApplication/python_scripts/dem_structures_coupling_gid_output.py:180]:
```suggestion
            kratos_variable = KratosMultiphysics.KratosGlobals.GetVariable(var)
```

File: applications/DemStructuresCouplingApplication/custom_utilities/dem_structures_coupling_utilities.h [hunk 2]
```diff
@@ -29,7 +29,7 @@
 #include "../../DEMApplication/custom_conditions/RigidFace.h"
 #include "../../DEMApplication/DEM_application_variables.h"
 #include "dem_structures_coupling_application_variables.h"
-
+#include "../../DEMApplication/custom_elements/spheric_continuum_particle.h"
```
Comment (philbucher) [inline applications/DemStructuresCouplingApplication/custom_utilities/dem_structures_coupling_utilities.h]: if you need DEM-stuffs then you could include it in the CMakeLists then you dont need the `../../DEMApplication`

Comment (maceligueta) [reply applications/DemStructuresCouplingApplication/custom_utilities/dem_structures_coupling_utilities.h]: Thanks @philbucher . Fixed.

File: applications/DemStructuresCouplingApplication/python_scripts/dem_structures_coupling_gid_output.py (hunk 1 above)
Comment (salvalatorre) [reply applications/DemStructuresCouplingApplication/python_scripts/dem_structures_coupling_gid_output.py:180]: Thanks, @philbucher, that's a good idea... We will implement it in a subsequent PR.

**Review by ipouplana — APPROVED** [review]
Thanks!

**Pull request merged**